src/composables/accounts.ts

```diff
@@ -39,12 +60,10 @@ const {
 } = createCallbackRegistry<(newAccount: IAccount, oldAccount: IAccount) => any>();
 
 const areAccountsRestored = ref(false);
+const isMnemonicRestored = ref(false);
+const encryptionData = ref<IEncryptionData | null>();
 
-/**
- * TODO Implement more safe way of storing mnemonic
- * For example by encrypting it with password or pin code.
- */
-const mnemonic = useStorageRef<string>(
+const mnemonic = useSecureStorageRef<string>(
```

Comment:
As far as I understood this state holds the encoded mnemonic, but after logging in the value changes to decoded one. So it actually is more like a computed value instead of state. In my opinion this is misleading as this behavior is different than regular `sorageRef` where the state reflects the local storage value.

My suggestion is to have `mnemonic` state to reflect 1:1 the state stored in the LocalStorage and another computed property e.g.: `mnemonicDecoded`).

What do you think @CedrikNikita ?

Reply:
I think proposal will align new mnemonic logic with the current approach.